dex/testing/loadbot/mantle.go

```diff
@@ -211,6 +211,7 @@ func (m *Mantle) orderMetered(ords []*orderReq, dur time.Duration) {
 	}
 	go func() {
 		ticker := time.NewTicker(dur / time.Duration(len(ords)))
+		defer ticker.Stop()
 		for {
 			select {
```

Comment:
New `Timer` every loop might be better than a `Ticker` too, but that's unrelated to your fix.